src/utils/user_utils.ts

```diff
@@ -150,6 +150,7 @@ export function filterProfilesMatchingTerm(users: Array<UserProfile>, term: stri
         const full = first + ' ' + last;
         profileSuggestions.push(first, last, full);
         profileSuggestions.push((user.nickname || '').toLowerCase());
+        profileSuggestions.push((user.position || '').toLowerCase());
         const email = (user.email || '').toLowerCase();
         profileSuggestions.push(email);
         profileSuggestions.push((user.nickname || '').toLowerCase());
```

Comment:
not part of your PR, but I wonder why are we adding twice the nickname. @mgdelacroix any ideas?

Reply:
Good catch! It's definitely not necessary, and as the function uses `some`, it does not alter the results, probably why it went unnoticed.

Addressed in https://github.com/mattermost/mattermost-redux/pull/1123

Reply:
thanks!